### ISSUE TITLE
Distance Not Normalized

### ISSUE THREAD

**Issue by guohaoxiang** [krrish94/chamferdist#14]
a = torch.zeros(21).view(1,-1,3)
b = a + 1
dist = chamferDist(a, b)

The expected distance between  a and b should be sqrt(3), however, chamferDist(a,b) returns 21. Is it because the distance not divided by the number of points?

**Comment by XueHu1995** [comment]
Same issue here

**Comment by Yihanhu** [comment]
[https://github.com/krrish94/chamferdist/blob/master/chamferdist/chamfer.py#L102](url)
There is a bug here.

**Comment by pansanity666** [comment]
The chamfer distance defined[ here ](http://graphics.stanford.edu/courses/cs468-17-spring/LectureSlides/L14%20-%203d%20deep%20learning%20on%20point%20cloud%20representation%20(analysis).pdf)is not normalized and is squared. I think it depends on  which kind of formulation you used.

**Comment by ghost** [comment]
For this issue, if each data(target and source) have 100 points, 
should we divide the result obtained by executing this code by 100?
